### PR TITLE
[TS SDK] Fix IndexerClient error parsing using JSON.stringify()

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Description
Hot fix for IndexerClient error parsing using JSON.stringify() to display the error message correctly on the console

image

### CODE REVIEW

**Review by jillxuu — APPROVED** [review]

**Review by banool — APPROVED** [review]

File: ecosystem/typescript/sdk/src/providers/indexer.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export class IndexerClient {
   private async queryIndexer<T>(graphqlQuery: GraphqlQuery): Promise<T> {
     const { data } = await axios.post(this.endpoint, graphqlQuery);
     if (data.errors) {
-      throw new Error(`Indexer data error ${data.errors}`);
+      throw new Error(`Indexer data error ${JSON.stringify(data.errors)}`);
```
Comment (movekevin) [inline ecosystem/typescript/sdk/src/providers/indexer.ts:40]: JSON.stringify can make the error very hard to read as it puts it all on a single line.

Comment (movekevin) [reply ecosystem/typescript/sdk/src/providers/indexer.ts:40]: Can we do something similar to what console.log does by default when you do console.log("Message", error)? It prints the object structure out

Comment (0xmaayan) [reply ecosystem/typescript/sdk/src/providers/indexer.ts:40]: `data.errors` is an array of objects, doing `console.error("msg",data.errors)` would print `msg [Object Object]`

Comment (movekevin) [reply ecosystem/typescript/sdk/src/providers/indexer.ts:40]: How about something like pretty print: https://www.npmjs.com/package/pretty-print-json

Comment (0xmaayan) [reply ecosystem/typescript/sdk/src/providers/indexer.ts:40]: I don't think this small thing requires to support a new package - more for us to maintain and more packages for our users to install also.
I'll work on a nicer solution though

Comment (0xmaayan) [reply ecosystem/typescript/sdk/src/providers/indexer.ts]: @movekevin updated with a screenshot

**Review by movekevin — APPROVED** [review]

**Pull request merged**